### ISSUE TITLE
Target "Alembic" links to target "Imath::Imath" but the target was not   found.

### ISSUE THREAD

**Issue by SoninnM** [alembic/alembic#341]
![image](https://user-images.githubusercontent.com/80504905/114966756-d11c0900-9ea5-11eb-9244-a71f4f1af8ea.png)

How to solve it? 
![image](https://user-images.githubusercontent.com/80504905/114966808-e98c2380-9ea5-11eb-894c-f01301c873b4.png)

**Comment by lamiller0** [comment]
I'm not sure about your Imath_DIR.

Maybe try this instead:
-DImath_ROOT=E:/SoftInstall/meshroom/plugins/Imath

You want to point it to the Imath location that has both the include and lib directory.

CMake stuff will usually be found under lib/cmake

**Comment by lamiller0** [comment]
Also you may have better luck with:

https://github.com/alembic/alembic/releases/tag/1.8.0_beta3

As I think you are using Imath 3.

**Comment by SoninnM** [comment]
> I'm not sure about your Imath_DIR.
> 
> Maybe try this instead:
> -DImath_ROOT=E:/SoftInstall/meshroom/plugins/Imath
> 
> You want to point it to the Imath location that has both the include and lib directory.
> 
> CMake stuff will usually be found under lib/cmake

Yes,I built Imath 3 on my pc.
When I change the value of Imath_DIR into E:/SoftInstall/meshroom/plugins/Imath or Add Entry Imath_ROOT=E:/SoftInstall/meshroom/plugins/Imath, error occured again as step2, it did not work.
I tried to build alembic-1.8.0_beta3 as the same steps above by downloading alembic-1.8.0_beta3.zip , the same error occured.
I have no idea what goes wrong ， it seems have nothing to do with alembic version.

**Comment by simogasp** [comment]
can you list the content of `E:/SoftInstall/meshroom/plugins/Imath/config`?

**Comment by SoninnM** [comment]
> can you list the content of `E:/SoftInstall/meshroom/plugins/Imath/config`?

CmakeLIsts.txt
Imath.pc.in
ImathConfig.cmake.in
ImathConfig.h.in
ImathSetup.cmake 
LibraryDefine.cmake
version,cmake

**Comment by simogasp** [comment]
you should point to the build directory where the actual config files are generated.
for example if you built the library in `E:/SoftInstall/meshroom/plugins/Imath/build` there inside you should see a directory named `config` containing the following files:
```
cmake_install.cmake
ImathConfig.cmake
ImathConfig.h
ImathConfigVersion.cmake
Imath.pc
ImathTargets.cmake
Makefile
```
if it is so, your `Imath_DIR` should point to `E:/SoftInstall/meshroom/plugins/Imath/build/config` .

**Comment by lamiller0** [comment]
After you built Imath 3 where did you install it?

i.e.

cmake --build . --target install

**Comment by SoninnM** [comment]
> Imath.pc
I did `set Imath_DIR=E:/SoftInstall/meshroom/plugins/Imath/build/config` on CMake UI above,you can see it.
I checked my build/config but I did not found the file `Imath.pc`. 
I have little programming experience.To find why it losed Imath.pc,I re-built and installed Imath in`plugins/Imath/build` twice.
1st checkout :
After `cmake .. -DCMAKE_INSTALL_PREFIX=E:/SoftInstall/meshroom/plugins/Imath/build` on developer command for vs2019, I open Imath.sln with VS2019, build ALL_BUILD and INSTALL. However, it does not have Imath.pc in `Imath\build\config`. So I delete the Imath\build file and tried again with your advice `cmake --build . --target install`
2nd checkout:
after `cmake .. -DCMAKE_INSTALL_PREFIX=E:/SoftInstall/meshroom/plugins/Imath/build` and `cmake --build . --target install`, it still does not contains imath.pc . 
I searched imath.pc in the directory` E:/SoftInstall/meshroom/plugins/Imath`, did not found Imath.pc but only `imath.pc.in` in Imath\config .  

I re-built alembic after each checkout, it still have the same error that can not links to target "Imath::Imath".
Does Imath.pc have anything to do with this error?

**Comment by simogasp** [comment]
no `Imath.pc` it's *nix thing (pkgconfig), it's not your problem.

If you install it in the build directory, then you should find a `bin` and a `lib` folder there inside
Then your path to give is `Imath_DIR:PATH=E:/SoftInstall/meshroom/plugins/Imath/build/lib/cmake/Imath/`.
You can check if the files are there inside.

As a side note and in general, I'd advise using a different install directory other than the build folder to avoid any possible conflict. When you build you can use e.g. `-DCMAKE_INSTALL_PREFIX=E:/SoftInstall/meshroom/plugins/Imath/build/install`
or whatever other directory you prefer (avoid spaces in names though) but not directly in the `build` folder.

**Comment by simogasp** [comment]
ops, sorry, my bad :(

**Comment by lamiller0** [comment]
This should be fixed in Alembic 1.8.1
